panopticum/static/js/graph-style.json

```diff
@@ -0,0 +1,99 @@
+[
```

Comment:
can we import that file and similar from https://cdnjs.com/libraries/cytoscape ?

Reply:
this file contains the custom styling specific for Panopticum, including the custom images

Reply:
What about other cytoscpape files? We can import in main HTML and not include to repository

Reply:
done